packages/core/storybook/Stamp.js

```diff
@@ -12,51 +34,18 @@ storiesOf('Stamp', module)
         'Use the <Stamp /> component to subtly display attributes alongside listing cells and on product detail pages. Use it in conjunction with an <Icon /> component to give it more context. An Icon placed within a Stamp will inherit the assigned Stamp color.'
     })(() => <Stamp>default stamp</Stamp>)
   )
-  .add('Small Stamp', () => (
-    <Stamp color="primary" size="small">
-      <Icon name="Pin" mr={1} />
-      Stamp Text
-    </Stamp>
-  ))
-  .add('Regular Stamp', () => (
-    <Stamp color="primary">
-      <Icon name="Pin" mr={1} />
-      Stamp Text
-    </Stamp>
-  ))
-  .add('Small Solid Stamp', () => (
-    <Stamp color="primary" size="small" variation="solid">
-      <Icon name="Pin" mr={1} />
-      Stamp Text
-    </Stamp>
-  ))
-  .add('Regular Solid Stamp', () => (
-    <Stamp color="primary" variation="solid">
-      <Icon name="Pin" mr={1} />
-      Stamp Text
-    </Stamp>
-  ))
   .add('All Colors', () => (
-    <div>
-      <Stamp color="text.light" mr={2}>
-        default stamp
-      </Stamp>
-      <Stamp color="primary" mr={2}>
+    <Cartesian
+      component={Stamp}
+      color={Object.keys(colors)}
+      variation={Object.keys(variations)}
+      size={Object.keys(sizes)}
+      m={3}
+    >
+      <>
```

Comment:
Is the fragment needed here?

Reply:
Yes, I guess Cardinal splits all the children into separate examples which means to have the examples as a single component you need just one child.